web/src/components/CustomMarquee/CustomMarquee.jsx

```diff
@@ -42,14 +42,11 @@ export default function CustomMarquee(props) {
         }
     }
 
-    let resizeTimout;
     function handleResize(){
         if(!resizeCooldown.current){
             resizeCooldown.current = true;
 
             assessMarquee()
-
-            resizeTimout = setTimeout(()=>{resizeCooldown.current = false;}, 1000) // set a cooldown for resize checks to avoid excessive renders
         }
```

Comment:
I removed unused code from our marquee component because I noticed it during the course of looking at the individual components that made up the player page stream title